dist/api_data.py

```diff
@@ -512,6 +512,11 @@ def __ge__(self, other):
             versions of WiredTiger. These operations are informational and
             skipped in recovery.''',
             type='boolean'),
+        Config('slow_checkpoint', 'false', r'''
+            if true, slow down checkpoint creation by slowing down internal 
+            page processing.
+            ''',
+            type='boolean')
```

Comment:
This needs to move to be listed alphabetically in the set of options, between `rollback_error` and `table_logging`.